packages/@webex/widget-cc-station-login/.babelrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  presets: [
+    ['@babel/preset-env', {targets: {node: 'current'}}],
```

Comment:
Do we want to restrict to some node version or allowing all the node environments available with the developer to use?

Reply:
Im open for discussion on this one.